### PR TITLE
add scripttest recipe

### PR DESCRIPTION
**Pull request by dmaticzka**

### CODE REVIEW

File: recipes/scripttest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  sha256: 33c2c5315538598652cecc28984557d55dbe75ff814e50e4307dc8fbfac51cd5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest
```
Comment (synapticarbors) [inline recipes/scripttest/meta.yaml]: Looking at the `setup.py` it does not seem like you need `pytest`.

File: recipes/scripttest/bld.bat [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pytest
```
Comment (synapticarbors) [inline recipes/scripttest/bld.bat]: Is there a reason why you are running `pytest` in the build script before anything is installed?  Typically any tests are run on the built package using the `test` section of the yaml file (although typically only "existence" import tests are run). If it is not necessary to run pytest, please delete the `bld.bat` and `build.sh` and use the `script` section of `build` as you did originally.

File: recipes/scripttest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{% set version = "1.3.0" %}
+
+package:
+  name: scripttest
+  version: "{{ version }}"
+
+source:
+  fn: scripttest-{{ version }}.tar.gz
+  url: https://github.com/pypa/scripttest/archive/{{ version }}.tar.gz
+  sha256: 33c2c5315538598652cecc28984557d55dbe75ff814e50e4307dc8fbfac51cd5
```
Comment (synapticarbors) [inline recipes/scripttest/meta.yaml]: We tend to prefer using a jinja variable for the `sha256`.

File: recipes/scripttest/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+test:
+  imports:
+    - scripttest
+
+about:
+  home: https://github.com/pypa/scripttest
+  license: MIT
+  license_family: MIT
+  license_file: docs/license.rst
+  summary: 'Helper to test command-line scripts.'
```
Comment (synapticarbors) [inline recipes/scripttest/meta.yaml]: Please add:

```
   dev_url: https://github.com/pypa/scripttest
   doc_url: https://scripttest.readthedocs.io
```

**Comment by dmaticzka** [comment]
Thanks for the comments! Running pytest during the build is not strictly necessary, I removed it as suggested.

**Pull request merged**